services/director/src/simcore_service_director/directorv2_proxy.py

```diff
@@ -0,0 +1,194 @@
+import logging
```

Comment:
not sure I understand that part.
Are you calling the director-v2 from director-v0? why?

Reply:
This was already answered below. I agree with your feedback. I will change how the services are called.